forums/forums-war/src/main/java/org/silverpeas/components/forums/control/ForumsSessionController.java

```diff
@@ -567,6 +568,10 @@ public boolean isAdmin() {
     return false;
   }
 
+  public boolean isAccessGuest() {
```

Comment:
This method should probably be removed according to suggested changes into `viewMessage.jsp`.

Reply:
OK, I will remove it.